[30;1m2023-04-01 21:16:54[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:16:54[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:16:54[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found rotterdam1965.discord.media
[30;1m2023-04-01 21:16:57[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1029 successfully terminated with return code of 0.
[30;1m2023-04-01 21:16:58[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1041036008516501554 (Guild ID 898910504737710120)
[30;1m2023-04-01 21:16:58[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:17:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:17:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:17:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found rotterdam1965.discord.media
[30;1m2023-04-01 21:17:24[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1041036008516501554 (Guild ID 898910504737710120)
[30;1m2023-04-01 21:17:24[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:17:24[0m [31mERROR   [0m [35mdiscord.player[0m Exception in voice thread Thread-301
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/player.py", line 698, in run
    self._do_run()
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/player.py", line 691, in _do_run
    play_audio(data, encode=not self.source.is_opus())
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/voice_client.py", line 683, in send_audio_packet
    self.socket.sendto(packet, (self.endpoint_ip, self.voice_port))
OSError: [Errno 9] Bad file descriptor[0m
[30;1m2023-04-01 21:17:25[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1036 has not terminated. Waiting to terminate...
[30;1m2023-04-01 21:17:25[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1036 should have terminated with a return code of -9.
[30;1m2023-04-01 21:17:55[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:17:55[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:17:56[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found rotterdam1965.discord.media
[30;1m2023-04-01 21:18:01[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1043 successfully terminated with return code of 0.
[30;1m2023-04-01 21:18:02[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1041036008516501554 (Guild ID 898910504737710120)
[30;1m2023-04-01 21:18:02[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:27:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:27:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:27:36[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found rotterdam1965.discord.media
[30;1m2023-04-01 21:27:40[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1057 has not terminated. Waiting to terminate...
[30;1m2023-04-01 21:27:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1041036008516501554 (Guild ID 898910504737710120)
[30;1m2023-04-01 21:27:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:27:40[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1057 should have terminated with a return code of -9.
[30;1m2023-04-01 21:27:53[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:27:53[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:27:54[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found rotterdam1965.discord.media
[30;1m2023-04-01 21:28:01[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1041036008516501554 (Guild ID 898910504737710120)
[30;1m2023-04-01 21:28:01[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:28:01[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1064 has not terminated. Waiting to terminate...
[30;1m2023-04-01 21:28:01[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1064 should have terminated with a return code of -9.
[30;1m2023-04-01 21:28:49[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 1 has successfully RESUMED session f06dca9621f20e3c16d88f9fa4d31580.
[30;1m2023-04-01 21:47:18[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:47:18[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:47:19[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east7761.discord.media
[30;1m2023-04-01 21:52:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:52:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:52:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south2966.discord.media
[30;1m2023-04-01 21:52:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1046034961595121674 (Guild ID 1043600587050209330)
[30;1m2023-04-01 21:52:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:53:00[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:53:00[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:53:00[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south2966.discord.media
[30;1m2023-04-01 21:53:06[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1046034961595121674 (Guild ID 1043600587050209330)
[30;1m2023-04-01 21:53:06[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:53:06[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1077 has not terminated. Waiting to terminate...
[30;1m2023-04-01 21:53:06[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 1077 should have terminated with a return code of -9.
[30;1m2023-04-01 21:53:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:53:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:53:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south2966.discord.media
[30;1m2023-04-01 21:53:28[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1046034961595121674 (Guild ID 1043600587050209330)
[30;1m2023-04-01 21:53:28[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:53:34[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:53:34[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:53:34[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south2966.discord.media
[30;1m2023-04-01 21:53:39[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1046034961595121674 (Guild ID 1043600587050209330)
[30;1m2023-04-01 21:53:39[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 21:53:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-01 21:53:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-01 21:53:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south2966.discord.media
[30;1m2023-04-01 21:54:17[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 1046034961595121674 (Guild ID 1043600587050209330)
[30;1m2023-04-01 21:54:17[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-01 22:05:50[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has successfully RESUMED session d829adf857064b3f9dca586bcc0fada9.
[30;1m2023-04-01 22:16:39[0m [34;1mINFO    [0m [35mdiscord.client[0m logging in using static token
[30;1m2023-04-01 22:16:41[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has connected to Gateway (Session ID: 804b89f16054306e3df81410fce79c08).
[30;1m2023-04-01 22:16:46[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 1 has connected to Gateway (Session ID: 51ba811346026d465ef6db447cf29508).
[30;1m2023-04-01 23:08:33[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 1 has successfully RESUMED session 51ba811346026d465ef6db447cf29508.
[30;1m2023-04-01 23:13:52[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has successfully RESUMED session 804b89f16054306e3df81410fce79c08.
[30;1m2023-04-01 23:45:10[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has successfully RESUMED session 804b89f16054306e3df81410fce79c08.
[30;1m2023-04-02 01:40:54[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 1 has successfully RESUMED session 51ba811346026d465ef6db447cf29508.
[30;1m2023-04-02 02:21:51[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has successfully RESUMED session 804b89f16054306e3df81410fce79c08.
[30;1m2023-04-02 03:01:14[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:01:14[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:01:14[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south171.discord.media
[30;1m2023-04-02 03:01:29[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 921962748798197780 (Guild ID 909259935030861844)
[30;1m2023-04-02 03:01:29[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:01:37[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:01:37[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:01:37[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-south171.discord.media
[30;1m2023-04-02 03:01:45[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 921962748798197780 (Guild ID 909259935030861844)
[30;1m2023-04-02 03:01:45[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:05:48[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:05:48[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:05:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:06:08[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:06:08[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:06:08[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 401 has not terminated. Waiting to terminate...
[30;1m2023-04-02 03:06:08[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 401 should have terminated with a return code of -9.
[30;1m2023-04-02 03:06:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:06:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:06:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:06:45[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:06:45[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:06:45[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 409 successfully terminated with return code of -9.
[30;1m2023-04-02 03:07:07[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:07:07[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:07:08[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:07:11[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 416 successfully terminated with return code of 0.
[30;1m2023-04-02 03:07:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:07:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:07:24[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:07:24[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:07:24[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:07:30[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:07:30[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:07:30[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 423 successfully terminated with return code of -9.
[30;1m2023-04-02 03:08:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:08:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:08:35[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:08:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:08:40[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:08:40[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 430 successfully terminated with return code of -9.
[30;1m2023-04-02 03:10:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:10:12[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:10:13[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:10:26[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:10:26[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:10:26[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 437 has not terminated. Waiting to terminate...
[30;1m2023-04-02 03:10:26[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 437 should have terminated with a return code of -9.
[30;1m2023-04-02 03:10:43[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:10:43[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:10:43[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:10:48[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:10:48[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:10:48[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 444 successfully terminated with return code of -9.
[30;1m2023-04-02 03:13:36[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Connecting to voice...
[30;1m2023-04-02 03:13:36[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Starting voice handshake... (connection attempt 1)
[30;1m2023-04-02 03:13:36[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Voice handshake complete. Endpoint found us-east5124.discord.media
[30;1m2023-04-02 03:13:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m The voice handshake is being terminated for Channel ID 974119893328855052 (Guild ID 974119892875878460)
[30;1m2023-04-02 03:13:49[0m [34;1mINFO    [0m [35mdiscord.voice_client[0m Disconnecting from voice normally, close code 1000.
[30;1m2023-04-02 03:13:49[0m [34;1mINFO    [0m [35mdiscord.player[0m ffmpeg process 451 successfully terminated with return code of -9.
[30;1m2023-04-02 03:15:59[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 144, in postTips
    await interaction.channel.send("**Tip:** Support the bot by upvoting on top.gg!\nhttps://top.gg/bot/84475719231353652!")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 738, in request
    raise Forbidden(response, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50013): Missing Permissions[0m
[30;1m2023-04-02 03:16:48[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:18:06[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:23:11[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:23:15[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:32:42[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:33:19[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:52:07[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:58:18[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_guild_join
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 133, in on_guild_join
    await channel.send(embed = em)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:59:33[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 03:59:37[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 04:00:33[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 04:10:11[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_message
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 204, in on_message
    await message.channel.send(eval(dict(data)['autoresponse_content'])[message.content])
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 680, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): too many 4xx response codes, try again later
[0m
[30;1m2023-04-02 04:17:05[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 165, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 140, in postTips
    await interaction.channel.send(f"**Tip:** {randomMsg}")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 738, in request
    raise Forbidden(response, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50001): Missing Access[0m
[30;1m2023-04-02 05:12:54[0m [34;1mINFO    [0m [35mdiscord.client[0m logging in using static token
[30;1m2023-04-02 05:12:55[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 0 has connected to Gateway (Session ID: 8a2d8b31a5e7ef99d5e9b0c4fb1a547f).
[30;1m2023-04-02 05:13:02[0m [34;1mINFO    [0m [35mdiscord.gateway[0m Shard ID 1 has connected to Gateway (Session ID: 60e1f82a9fc4ac2542191515da57e33f).
[30;1m2023-04-02 05:15:10[0m [31mERROR   [0m [35mdiscord.client[0m Ignoring exception in on_interaction
[31mTraceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "main.py", line 166, in on_interaction
    await postTips(interaction)
  File "/home/runner/bot/other/utilities.py", line 144, in postTips
    await interaction.channel.send("**Tip:** Support the bot by upvoting on top.gg!\nhttps://top.gg/bot/84475719231353652!")
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/abc.py", line 1561, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/http.py", line 738, in request
    raise Forbidden(response, data)
discord.errors.Forbidden: 403 Forbidden (error code: 50001): Missing Access[0m
